tests/models/test_hf_model.py

```diff
@@ -594,17 +575,12 @@ def test_separate_eval_metrics(tiny_bert_model, tiny_bert_tokenizer):
 def test_add_eval_metrics(tiny_bert_model, tiny_bert_tokenizer):
```

Comment:
this test is actually the same as the one above it now, and you can just drop it